server/server.go

```diff
@@ -218,6 +219,12 @@ func (svr *Server) HandlePut(w http.ResponseWriter, r *http.Request) (commitment
 	commitment, err := svr.router.Put(r.Context(), meta.Mode, comm, input)
 	if err != nil {
 		err = fmt.Errorf("put request failed with commitment %v (commitment mode %v): %w", comm, meta.Mode, err)
+
+		if utils.ErrorMessageContainsAny(err, store.OversizedEigenDAError, store.OversizedProxyError) {
+			svr.WriteBadRequest(w, err)
```

Comment:
nit unrelated to this PR but I don't like that right now we double log every error:
1. once in the svr.WriteBadRequest/InternalError/etc
2. second time in the logger middleware

Would be good to find a nice way to clean this up, and ideally only print in the logging middleware, including the correct status. Moving to a framework like https://echo.labstack.com/docs/middleware/logger is also a possibility.

Reply:
think we can achieve this quickly by just printing in the logging middleware directly and removing the logs from the writer functions - but a fancy middleware wrapper would be nice hehe